### ISSUE TITLE
Feature request: Option settings to disable features other than syntax.

### ISSUE THREAD

**Issue by LumaKernel** [ocamllabs/vscode-ocaml-platform#889]
Hello, thank you for great extension. I'm glad to see this in open-vsx.org, it helps me so much.
I want to suggest implementing a new options to control language servers. This feature will help when we only want to use syntax highlighting.

**Comment by ulugbekna** [comment]
Reminds me of https://github.com/ocamllabs/vscode-ocaml-platform/issues/435

easy solution mentioned in that thread: remove ocamllsp

you can also rename ocamllsp in your sandbox to something else, and rename back when you want it used

**Comment by LumaKernel** [comment]
> easy solution mentioned in that thread: remove ocamllsp

I tried this and published to open-vsx.org. It's working for me. https://open-vsx.org/extension/luma/ocaml-platform-syntax

**Comment by LumaKernel** [comment]
> It's fairly easy, but it's not something that we can spend time on. It's too specialized of a request that is scarcely useful to everyone else who finds the diagnostics useful.

I agree this in that issue. Someone like me should work on this, but I want this to be stayed open.
I'm using this ext without ocaml environment and using only for Jupyter Notebook's highlight so it every time shows me like _there is no ocaml something..._
